test/jdk/java/foreign/TestLayoutPaths.java

```diff
@@ -122,10 +122,15 @@ public void testIncompleteAccess() {
         seq.varHandle(sequenceElement());
     }
 
+    public void testByteOffsetHandleRange() {
```

Comment:
Missing `@Test`?

Reply:
FWIW, it looks like this needs `@Test(expectedExceptions = IllegalArgumentException.class)` since range elements are not allowed for `byteOffsetHandle`

Reply:
Note sure - javadoc says:
```
* @throws IllegalArgumentException if the layout path is not <a href="#well-formedness">well-formed</a> for this layout.
     * @throws IllegalArgumentException if the layout path contains one or more <a href=#deref-path-elements>dereference path elements</a>.
```

Which seems correct. The handle version can accept indices, so that's ok?

Reply:
Note that the javadoc has been rectified here (as part of the PR) as I think it has always been wrong in the past (probably cut and paste issue from the non-MH version?)

Reply:
But, the implementation of `byteOffsetHandle` rejects range elements: https://github.com/openjdk/jdk/pull/14098/files/265e0d5b082ad54d1cadea1d4a8bd844e72e3926#diff-90ecdacbee63a90f6e695a8bb4c1ad69b591602501613a774561c4032f01fb95R352

So this test fails (at least it does here...)

Reply:
I see, the issue is with the impl (I agree) not with the javadoc as I understood.